packages/mobile/src/paymentRequest/OutgoingPaymentRequestListScreen.tsx

```diff
@@ -44,66 +40,43 @@ const mapStateToProps = (state: RootState): StateProps => ({
   recipientCache: recipientCacheSelector(state),
 })
 
-type Props = WithNamespaces & StateProps & DispatchProps
-
-export class OutgoingPaymentRequestListScreen extends React.Component<Props> {
-  static navigationOptions = () => ({
-    ...headerWithBackButton,
-    headerTitle: i18n.t('paymentRequestFlow:outgoingPaymentRequests'),
-  })
-
-  renderRequest = (request: PaymentRequest, key: number, allRequests: PaymentRequest[]) => {
-    const { recipientCache } = this.props
-    const requester = getRecipientFromPaymentRequest(request, recipientCache)
+type Props = NavigationInjectedProps & WithNamespaces & StateProps & DispatchProps
 
-    return (
-      <View key={key}>
-        <OutgoingPaymentRequestListItem
-          id={request.uid || ''}
-          amount={request.amount}
-          updatePaymentRequestStatus={this.props.updatePaymentRequestStatus}
-          updatePaymentRequestNotified={this.props.updatePaymentRequestNotified}
-          requester={requester}
-          comment={request.comment}
-        />
-        {key < allRequests.length - 1 && <View style={styles.separator} />}
-      </View>
-    )
-  }
+export const listItemRenderer = (params: {
```

Comment:
I this `props` is a more conventional name for this var. Or you can use desctructuring and avoid the name entirely

Reply:
I wanted to avoid confusion with props in React component and desctructuring would trigger `no-shadow` eslint rules. So, I've came with `params`.

Reply:
Ok